cmd/createenr.go

```diff
@@ -59,7 +59,7 @@ func runCreateEnrCmd(w io.Writer, config p2p.Config, dataDir string) error {
 	}
 	defer db.Close()
 
-	_, _ = fmt.Fprintf(w, "Created ENR private key: %s/p2pkey\n", dataDir)
+	_, _ = fmt.Fprintf(w, "Created ENR private key: %s/charon-enr-private-key\n", dataDir)
```

Comment:
nit: can shorten to `enr-private-key`

Reply:
We went with `charon-enr-private-key` so I think it's accurate as is.